src/app/submit-item/components/submit/submit.component.html

```diff
@@ -4,7 +4,7 @@
       <h1><span class="alt">Fall einreichen.</span></h1>
       <div>
         Gib hier die Details zu deinem Fall ein. Das kann entweder eine Behauptung sein, oder ein Link. Sobald du auf
-        Fall absenden klickst, prüfen wir ob wir diesen Fall schon bearbeitet haben. Dann erhältst du sofort ein Ergebnis.
+        Fall absenden klickst, prüfen wir, ob wir diesen Fall schon bearbeitet haben. Dann erhältst du sofort ein Ergebnis.
         Falls dein Fall neu ist, werden wir ihn sofort an unser Team von Detektiven weitergeben, die den Fall dann für
         dich prüfen.
```

Comment:
No comma within this sentence:
"Das kann entweder eine Behauptung sein, oder ein Link"  ->
"Das kann entweder eine Behauptung oder ein Link sein."